content/docs/releases/enterprise/install/quickstart.mdx

```diff
@@ -1,59 +1,41 @@
 ---
 # cSpell:ignore pygpgme makecache disablerepo enablerepo
 
-title: Enterprise Console Quickstart
+title: Run Pomerium Enterprise With Docker
 sidebar_label: Quickstart
 description: Demo Pomerium Enterprise
 sidebar_position: 1
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import HAConfig from '../../../../examples/enterprise/hosted-auth-config.yaml.md';
+import HACompose from '../../../../examples/enterprise/hosted-auth-docker.yaml.md';
 
-# Pomerium Enterprise Quickstart
+# Run Pomerium Enterprise With Docker
 
-Learn how to run Pomerium Enterprise using Docker Compose.
+Run Pomerium Enterprise with Docker containers and connect to the Console.
+
+This guide uses our Hosted Authenticate Service. If you use our hosted service and have a license key, you can complete this guide in **under 5 minutes**.
+
+See the Self-Hosted Authenticate Service page if you want to self-host Pomerium.
```

Comment:
Link :)

Reply:
The link isn't available in this PR. I just merged the HA docs in a separate PR. I planned on linking after the HA docs are merged.